### DISCUSSION TITLE
建议将右键菜单新建文档的固定四项去除

### DISCUSSION THREAD

**Discussion by floatingrain** [linuxdeepin/developer-center#2844] [Features Request & Ideas | 特性请求 & 头脑风暴]
通过命令行搜索并且删除了/usr某目录下（忘了具体路径）新建办公文档等四个模板一共八个文件，然而发现右键菜单里依然停留老四项，通过浏览代码发现这四项是硬编码的，完全没有给用户自定义的空间。建议从代码中删除硬编码，改为类似扫描~/.Templete模板目录的形式，这样不失普通用户的易用性，也能满足高级用户自定义。顺便：模板都还是doc等二进制格式，这个也得改成新的docx等格式吧。

**Comment by JiaOkay** [comment]
你说的是这个，不要可以自行删除，不过官方确实应该更新下了
![图片](https://user-images.githubusercontent.com/38833358/156685723-5e0ec55a-3aa2-48ed-933a-10e44a683164.png)


用户目录下支持自定义：

![图片](https://user-images.githubusercontent.com/38833358/156685188-8e338d35-6cdb-4acd-917e-91c6e4e424de.png)


效果
![图片](https://user-images.githubusercontent.com/38833358/156685235-143f2661-bf1f-4657-88dc-0b8c43bbd11f.png)

**Comment by meiyixiang** [comment]
41/5000 
This is good advice, but we need to think about how to provide developers with more convenient features like this.

**Comment by meiyixiang** [comment]
你好你的建议，我们已记录并会纳入后期产品规划

**Reply by floatingrain** [reply]
你好，该规划至今还没有任何改动，请尽快纳入开发流程。陈旧的四项显然已经让右键菜单“新建文档”条目失去了意义
